clang/test/3C/malloc_array.c

```diff
@@ -20,7 +20,7 @@ void bar(void) {
   //CHECK: y = (int *)5;
   int *z = foo(y);
   //CHECK_NOALL: _Ptr<int> z = foo(y);
-  //CHECK_ALL: _Ptr<int> z = foo(_Assume_bounds_cast<_Array_ptr<int>>(y,  count(2 + 1)));
+  //CHECK_ALL: _Ptr<int> z = foo(_Assume_bounds_cast<_Array_ptr<int>>(y, bounds(unknown)));
```

Comment:
This change is expected and acceptable based on our discussions, but it is also one we could easily improve in the future. Since the bound is a constant expressions we *shouldn't* need worry about the subtle scoping issue associated with bounds using variables.

Reply:
Is it worth posting a regression issue?

Reply:
I'll add a follow up comment on #524